src/widgets/block.rs

```diff
@@ -507,5 +532,39 @@ mod tests {
                 height: 0,
             },
         );
+        assert_eq!(
```

Comment:
The `title_alignment` is not taken into account in `Block::inner`. Therefore, is there a reason to add those test cases here ?

Reply:
I added them to show that `title_alignment` does not effect `Block::inner`. I am not sure, if that is necessary.